swift/read_json_file.swift

```diff
@@ -0,0 +1,18 @@
+//Read JSON file, transform and print to stdout
+import Foundation
+
+guard let jsonFile = CommandLine.arguments.dropFirst().first else {
+    print("Usage: swift script.swift <json_file>")
+    exit(1)
+}
+
+let fileURL = URL(fileURLWithPath: jsonFile)
+let jsonData = try Data(contentsOf: fileURL)
+let people = try JSONSerialization.jsonObject(with: jsonData) as? [[String: Any]] ?? []
```

Comment:
```suggestion
let people = try JSONSerialization.jsonObject(with: jsonData) as! [[String: Any]]
```

Let's fail immediately if we don't get the type we expect